src/SFML/Graphics/Text.cpp

```diff
@@ -212,7 +212,8 @@ void Text::setFillColor(const Color& color)
                 if (m_verticesBuffer.getVertexCount() != m_vertices.getVertexCount())
                     m_verticesBuffer.create(m_vertices.getVertexCount());
 
-                m_verticesBuffer.update(&m_vertices[0]);
+                if (m_vertices.getVertexCount())
```

Comment:
We usually write "> 0" explicitly for this kind of tests.

Reply:
Fixed.